vpdq/python/vpdq.pyx

```diff
@@ -52,23 +52,23 @@ class VpdqFeature:
     frame_number: int
     hash: Hash256
     hex: str
-    timestamp: double
+    timestamp: float
```

Comment:
These typehint changes are warnings I spotted in the output of the Cython build and fixed.

Reply:
I couldn't figure out how to actually run any of the typehints on .pyx files, so much appreciated.